mtags/src/main/scala-3/scala/meta/internal/pc/completions/CompletionProvider.scala

```diff
@@ -261,6 +261,10 @@ class CompletionProvider(
                   mkItem(
                     "{" + sym.fullNameBackticked + completionTextSuffix + "}"
                   )
+                case _ if v.completionItemKind == CompletionItemKind.Method =>
```

Comment:
I don't think that it never makes sense for methods to put the full name:
```scala
//> using scala 3.3.1

object O {
  def mmm = 3
  class Test {
    val mmm = "aaaa"
    val foo = mmm@@
  }
}
```

Reply:
Yeah, it seems that the issue is that we don't see the extension method in the scope and we don't shorten the name. Especially this doesn't make sense for extension methods.

Reply:
> I don't think that it never makes sense for methods to put the full name

Thanks, I missed this case would also be changed, I changed it so the change applies only to extension methods.